acceptance-tests/tests/workflows/accountManagementFlow.spec.ts

```diff
@@ -68,17 +56,9 @@ describe('Account Management Flow', () => {
         'remove',
         `--account=${accountName}`,
       ]);
-    });
-  });
-
-  describe('hs accounts list', () => {
-    it('should not list the removed authenticated account', async () => {
-      const output = await testState.cli.executeWithTestConfig([
-        'accounts',
-        'list',
-      ]);
 
-      expect(output).not.toContain(accountName);
```

Comment:
Why did we remove this?

Reply:
I misunderstood the nature of the tests 😬 I can change back. Insert the Homer Simpson disappearing GIF here please.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1123/commits/4d29cfa5d53a3cd02049061caea52a460d5ef11f